test/e2e/secrets_test.go

```diff
@@ -182,8 +182,9 @@ func testPrivateKeyChange(t *testing.T) {
 	require.NoError(t, err, "operator Upgradeable condition not in proper state")
 
 	// Re-create the known private key so SSH connection can be re-established
```

Comment:
This comment is in the incorrect spot now.

Reply:
The sequence is
1) loading existing nodes
2) changing the private-key
3) ensuring that new nodes are created is part of this process.

I can either remove the comment or break this process out into its own function.

Reply:
I am fine if you move this into a new method